core/src/main/resources/static/js/requestAcls.js

```diff
@@ -595,14 +595,19 @@ app.controller("requestAclsCtrl", function($scope, $http, $location, $window) {
                 for (var i = 0; i < sURLVariables.length; i++)
                 {
                     var sParameterName = sURLVariables[i].split('=');
+                    
                     if (sParameterName[0] == "topicname")
                     {
                         $scope.topicSelectedFromUrl = sParameterName[1]
                         $scope.addAcl.topicname = $scope.topicSelectedFromUrl;
 
                         $scope.getAllTopics("ALL");
                         $scope.getTopicTeam($scope.addAcl.topicname);
-                    }
+                    } else if (sParameterName[0] == "?envId" || sParameterName[0] == "envId")
```

Comment:
```suggestion
                    } else if (sParameterName[0] == "envId")
```

this should be ok?

Reply:
Yes I see what i did wrong building the url